hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieWriteHandle.java

```diff
@@ -273,6 +279,33 @@ protected HoodieFileWriter createNewFileWriter(String instantTime, Path path, Ho
     return HoodieFileWriterFactory.getFileWriter(instantTime, path, hoodieTable, config, schema, taskContextSupplier);
   }
 
+  protected HoodieLogFormat.Writer createLogWriter(
```

Comment:
this meant for CDC log writing right? can we move it to CDC-dedicated classes instead of mixing in write handle. people may be confused about when/what scenario to use this.

Reply:
not just used in CDC. HoodieAppendHandle also will use this method to create the log writer. I just move it to the base class, so that it can be reused.